### DISCUSSION TITLE
Harp Behavior software

### DISCUSSION THREAD

**Discussion by petersaj** [harp-tech/protocol#74] [Open-ended]
We installed the Harp Behavior software (v.2.0.0) from this website: https://www.cf-hw.org/harp/behavior

When we try to open it, we an error asking for the LabVIEW 2010 runtime engine:
![image](https://user-images.githubusercontent.com/17985056/212961229-ec5f4b59-e4ef-4afa-aac6-299ddfb43421.png)

Downloading the newest LabVIEW runtime engine doesn't solve this issue, and the 2010 runtime engine is not available for download. What should we do to run the Harp Behavior software?

**Comment by filcarv** [comment]
The introductory information of the [Harp Behavior's repository](https://github.com/harp-tech/device.behavior) has the necessary information and links under the topic **Interface**.

Before installing, I would recommend uninstalling any other LabVIEW runtime and rebooting the computer.
